echo-notifications/src/main/groovy/com/netflix/spinnaker/echo/slack/SlackNotificationService.groovy

```diff
@@ -51,12 +51,15 @@ class SlackNotificationService implements NotificationService {
   EchoResponse.Void handle(Notification notification) {
     def text = notificationTemplateEngine.build(notification, NotificationTemplateEngine.Type.BODY)
     notification.to.each {
+      def response
```

Comment:
prefer actual type here
```suggestion
      Response response
```

Reply:
Cool 'nuff -- I'm not super familiar with Groovy, I just mimick'd the "def text" variable definition earlier in the function.